src/main/java/com/digitalasset/testing/cucumber/steps/LedgerInteractions.java

```diff
@@ -44,7 +46,7 @@
   private SandboxManager sandboxManager;
   private static final Logger logger = LoggerFactory.getLogger(LedgerInteractions.class);
 
-  public LedgerInteractions() {
+  public LedgerInteractions(World world) {
```

Comment:
Just one thing: don't you think naming it as "Config" for example (or something different than World) would be better?

Reply:
We can rename it for sure. However, both are too broad terms. I would rather use something specific. Maybe SandboxConfig.